Misc/NEWS.d/next/Documentation/2021-07-03-18-25-17.bpo-44558.0pTknl.rst

```diff
@@ -0,0 +1,2 @@
+Match the docstring and python implementation of :func:countOf to the behaviour

```

Comment:
```suggestion
Match the docstring and python implementation of :func:`~operator.countOf` to the behavior

```

Reply:
Thank you for looking at this! Since you have made https://github.com/python/cpython/pull/27012, I will close this pull request.